src/watchGuard.ts

```diff
@@ -0,0 +1 @@
+export = require('typescript/lib/watchGuard.js')
```

Comment:
Can you fix it to es6 import

Reply:
Are imports ok? May be resolve modules dynamically via resolve module, like in src/tsc.ts ?